ibm_cloud_sdk_core/base_service.py

```diff
@@ -35,6 +35,10 @@
                     cleanup_values, read_external_sources, strip_extra_slashes)
 from .version import __version__
 
+
+logger = logging.getLogger(__name__)
```

Comment:
Please move this line under the uncommented import block and surround it 2 blank lines, like this:
```python
from .version import __version__

# Uncomment this to enable http debugging
# import http.client as http_client
# http_client.HTTPConnection.debuglevel = 1


logger = logging.getLogger(__name__)


#pylint: disable=too-many-instance-attributes
#pylint: disable=too-many-locals
class BaseService:
```